### PR TITLE
Fix/82 featuredwork

### PR DESCRIPTION
**Pull request by HarryuCATGREY**
### Descriptions
##Things I did
1. Merge PicturedBox into MemberCard
2. Fix the paragraph in FeaturedWork component.
3. Use public button in FeaturedWork component.
4. Folder structure img into images

- Link to UI design (if possible):
image

### Provide screenshots or videos that the function is working.
image
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

File: components/base/MemberCard.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+type CardDirectionVariant = "column" | "column_reverse" | "row";
+type CardSizeVariant = "lg" | "md" | "sm";
+
+interface CardProps {
+  title: string;
+  subtitle: string;
+  image: string;
+  direction?: CardDirectionVariant;
+  size?: CardSizeVariant;
+}
```
Comment (tatoMa) [inline components/base/MemberCard.tsx:13]: Nice work!

**Review by tatoMa — APPROVED** [review]
LGTM

File: components/general/FeaturedWork.tsx [hunk 2]
```diff
@@ -35,40 +36,60 @@ export default function FeaturedWork() {
...
+
+          {/* {imgBoxes.map((box) => (
             <Picturedbox
               key={box.id}
               location={box.location}
               title={box.title}
               hasBorder={box.hasBorder}
             ></Picturedbox>
-          ))}
+          ))} */}
```
Comment (LULIU-9) [inline components/general/FeaturedWork.tsx:92]: Remove unused comments.

**Review by LULIU-9 — APPROVED** [review]

File: components/general/FeaturedWork.tsx (hunk 2 above)
Comment (HarryuCATGREY) [reply components/general/FeaturedWork.tsx:92]: OK

**Pull request merged**